### ISSUE TITLE
Why does wslconfig.exe /l still list "WLinux": Answered

### ISSUE THREAD

**Issue by ghost** [WhitewaterFoundry/Pengwin#428]
I'm a new user that bought Pengwin just today so there's not any remnants of a WLinux upgrade to speak of.

I was changing my default linux from Debian to Pengwin (I'm keeping Debian until I get all the stuff I use on Debian installed on Pengwin) so I ran `wslconfig /l` and it says WLinux instead of Pengwin.

So you might want to change that to Pengwin at some point.

**Comment by crramirez** [comment]
Sure thank you for reporting

**Comment by sirredbeard** [comment]
This is not an oversight, it's a limitation in WSL.

Changing the unique WSL identifier registered in WSL would have broken everyone's existing installation.

They would have been left with a cryptic error code, unable to access their files, and having to reset the app.

That was simply out of the question.

We stuck with WLinux until we can gracefully modify that. I am going to talk to the Microsoft team about it.

The other option was to discontinue WLinux altogether and launch a new Pengwin app, offering it for free to existing WLinux users.

Ultimately I decided to upgrade everyone in place, including to the new apt repo all at one time.

We've had some issues but overall I think it went really well.

Some Taskbar glitches we've got an easy workaround for, a minor issue for folks on really old builds of Windows we are fixing, a minor issue for folks with really old versions of WLinux we fixed.

So far no reports of data loss or app breakage. Just some people who don't follow us on Twitter or the blog who were...surprised.
